packages/insured-bridge-relayer/src/CrossDomainFinalizer.ts

```diff
@@ -65,13 +66,20 @@ export class CrossDomainFinalizer {
     let nonce = await this.l2Client.l2Web3.eth.getTransactionCount(this.account);
     for (const l2Token of bridgeableL2Tokens) {
       // For each bridgeable L2Token, check the balance in the deposit box. If it is greater than
-      // crossDomainFinalizationThreshold, as a percentage, then we can bridge it.
+      // crossDomainFinalizationThreshold, as a percentage, then we can bridge it. If the liquidity utilization is
+      // greater than 75% then half the crossDomainFinalizationThreshold to send funds more aggressively over the
+      // bridge at high utilizations.
 
       try {
         const { symbol, decimals, l2PoolBalance } = await this._getL2TokenInfo(l2Token);
         const l1PoolReserves = await this._getL1PoolReserves(l2Token);
+        const l1PoolUtilization = await this._getL1PoolUtilization(l2Token);
 
-        if (l2PoolBalance.gt(toBN(this.crossDomainFinalizationThreshold).mul(l1PoolReserves).div(toBN(100)))) {
+        const scaledCrossDomainFinalizationThreshold = l1PoolUtilization.gt(toBNWei("0.75"))
```

Comment:
should this 0.75 value be parameterized?

Reply:
I'll approve but I think its somewhat likely we realize 0.75 threshold is too slow/too fast and we'll have to annoyingly wait for CI to pass to change the value so if you have time I would just parameterize this so can we modify it easier with a config change